src/main/webapp/WEB-INF/pages/cart.jsp

```diff
@@ -0,0 +1,103 @@
+<%@ page contentType="text/html;charset=UTF-8" %>
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ taglib prefix="fmt" uri="http://java.sun.com/jsp/jstl/fmt" %>
+<%@ taglib prefix="tags" tagdir="/WEB-INF/tags" %>
+
+<jsp:useBean id="cart" type="com.es.phoneshop.model.cart.Cart" scope="request"/>
+
+<tags:master pageTitle="Cart">
+<p>
+    <c:if test="${not empty param.modalSuccess}">
+        <h3 class="success">
+            ${param.modalSuccess}
+        </h3>
+    </c:if>
+    <c:if test="${not empty errors}">
+        <h3 class="error">
+            There was some errors updating cart
+        </h3>
+    </c:if>
+</p>
+    <c:if test="${cart.getItems().size() > 0}">
+    <form method="post" action="${pageContext.servletContext.contextPath}/cart">
+    <table>
+        <thead>
+          <tr>
+            <td>Image</td>
+            <td>
+                Description
+            </td>
+            <td class="price">
+                Price
+            </td>
+            <td>
+                Quantity
+            </td>
+          </tr>
+        </thead>
```

Comment:
Please format the HTML code:
```
<form method="post" action="${pageContext.servletContext.contextPath}/cart">
    <table>
        <thead>
            <tr>
                <td>Image</td>
                <td>Description</td>
                <td class="price">Price</td>
                <td>Quantity</td>
            </tr>
        </thead>
```

More info: https://www.w3schools.com/html/html5_syntax.asp